docs/changelog.rst

```diff
@@ -1,5 +1,17 @@
 Changelog
 ================================
+CBC SDK 1.3.1 - Released June 15, 2021
+--------------------------------
+
+New Features:
+
+* Allow the SDK to accept a pre-configured ``Session`` object to be used for access, to get around unusual
+  configuration requirements.
+
+Bug Fixes:
+
+* Fix to code for adding a new access profile to a user access grant.
```

Comment:
Can you be more specific as to what was fixed? `Fix to code` is not very descriptive

Reply:
Yes, how's this?